app/demoservice/libs/demos.py

```diff
@@ -7,13 +7,14 @@
 import shutil
 import socket
 import yaml
+from distutils.version import StrictVersion
+from subprocess import Popen, check_output
 from django.conf import settings
 from github3 import login
-from subprocess import Popen
 
 from demoservice.logging import get_demo_logger
 
-
+MIN_RUNSCRIPT_VERSION = '2.0.0'
```

Comment:
Not sure about this version, we might want to go higher?

Reply:
Version 2 seems pretty safe